aicsimageio/tests/readers/reader_test_utils.py

```diff
@@ -43,14 +43,17 @@ def run_image_read_checks(
     ReaderClass: Reader,
     uri: types.PathLike,
     set_scene: int,
-    expected_scenes: Set[int],
+    expected_scenes: Tuple[int],
```

Comment:
Just to point it out each member of a set is unique - ie no redundancy. Tuple has no such property so conceivably one could request a scene multiple times.

Reply:
Huh. I remember making this change because for some reason I thought I couldn't iterate over sets, i.e. `for scene_id in reader.scenes:` but I just gave it a try and it definitely works so will revert this change.

Reply:
I remember why this has to be a `Tuple`, `Sets` in Python are not subscriptable. I.E. you can't call `self.scenes[0]` :upside_down_face: